### PR TITLE
[SPARK-25496][SQL][followup] avoid using to_utc_timestamp

### PR DESCRIPTION
**Pull request by cloud-fan**
## What changes were proposed in this pull request?

in https://github.com/apache/spark/pull/24195 , we deprecate `from/to_utc_timestamp`.

This PR removes unnecessary use of `to_utc_timestamp` in the test.

## How was this patch tested?

test only PR

### CODE REVIEW

**Comment by cloud-fan** [comment]
cc @MaxGekk @srowen

File: sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala [hunk 1]
```diff
@@ -342,55 +342,52 @@ class StreamingAggregationSuite extends StateStoreMetricsTest with Assertions {
...
-        CheckLastBatch((90L, 1), (100L, 1), (105L, 1))
-      )
-    }
+    import testImplicits._
+    val clock = new StreamManualClock
+    val tz = TimeZone.getDefault.getID
+    val inputData = MemoryStream[Long]
+    val aggregated =
+      inputData.toDF()
+        .select(($"value" * DateTimeUtils.SECONDS_PER_DAY).cast("timestamp").as("value"))
```
Comment (srowen) [inline sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala]: This is basically the only change right? can you remove `tz` too?
Seems OK, especially if pulling timezones out of the equation here.

**Review by MaxGekk — APPROVED** [review]

File: sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala (hunk 1 above)
Comment (cloud-fan) [reply sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala]: there is one more: I replaced `current_date` with `current_timestamp().cast("date")`.

`current_date` returns date in UTC. We can't compare it with the value of casting timestamp column to date. There is a timezone shift when casting between date and timestamp.

**Review by attilapiros — COMMENTED** [review]
LGTM

Tip for the next reviewer: after switching on "Hide whitespace changes" the diff is quite small and easier to review.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Pull request closed without merging**